app/Keycloak/Repositories/EloquentKeycloakClientRepository.php

```diff
@@ -78,7 +79,9 @@ public function getMissingRealmsByIntegrationId(UuidInterface $integrationId): R
     {
```

Comment:
If you make this return the missing environments you make the code much easier. Because inside the `handleCreatingMissingClients` you have the list of realms and you just search for the missing inside the already constructed list.